configuration/igniteVersionToDisplayName.properties

```diff
@@ -1 +1 @@
-1.3-SNAPSHOT=1.3-SNAPSHOT (Fuse Ignite TP4)
+1.3.10.fuse-000001-redhat-1=1.3.10.fuse-000001-redhat-1 (Fuse Ignite 7.0 GA)
```

Comment:
its not really Fuse Ignite 7.0 GA, no? Shouldn't it be more like (Fuse 7.0.0 GA) instead?

Reply:
i think we should merge as-is, otherwise we might miss the code freeze of today and will be able to update it only on 5 or 6 June

Reply:
proposals:
![image](https://user-images.githubusercontent.com/1105127/40421685-0655e30e-5e8d-11e8-
88e9-3aace0388141.png)
![image](https://user-images.githubusercontent.com/1105127/40421701-139d8ef4-5e8d-11e8-8a54-5374a2e5519c.png)